MAUI/DataForm/floating-label-layout.md

```diff
@@ -0,0 +1,777 @@
+        <dataForm:DataFormTextItem.LeadingView>
+            <Label Text="F" FontSize="18" TextColor="Gray" FontFamily="InputLayoutIcons" HeightRequest="24" VerticalTextAlignment="End" />
+        </dataForm:DataFormTextItem.LeadingView>
+    </dataForm:DataFormTextItem>
+</dataForm:SfDataForm.Items>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight XAML %}
+<ContentPage 
+...
+xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+<dataForm:SfDataForm x:Name="dataForm" 
+                        LayoutType="TextInputLayout"
+                        DataObject="{Binding ContactInfo}">
+<dataForm:SfDataForm.Items>
+    <dataForm:DataFormTextItem FieldName="Name" 
+                                ShowTrailingView="True" 
+                                TrailingViewPosition="Inside">
+        <dataForm:DataFormTextItem.TrailingView>
+            <Label Text="F" FontSize="18" TextColor="Gray" FontFamily="InputLayoutIcons" HeightRequest="24" VerticalTextAlignment="End" />
+        </dataForm:DataFormTextItem.TrailingView>
+    </dataForm:DataFormTextItem>
+</dataForm:SfDataForm.Items>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.ShowLeadingView = true;
+        e.DataFormItem.LeadingViewPosition = TextInputLayoutViewPosition.Outside;
+        e.DataFormItem.LeadingView = new Label { Text = "F", FontSize = 18, TextColor = Colors.Gray, FontFamily = "InputLayoutIcons", HeightRequest = 24, VerticalTextAlignment = TextAlignment.End };
+    }
+}
+
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.ShowTrailingView = true;
+        e.DataFormItem.TrailingViewPosition = TextInputLayoutViewPosition.Inside;
+        e.DataFormItem.TrailingView = new Label { Text = "F", FontSize = 18, TextColor = Colors.Gray, FontFamily = "InputLayoutIcons", HeightRequest = 24, VerticalTextAlignment = TextAlignment.End };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Enable password visibility toggle for password editor
+Password toggle visibility in floating label layout is used to show or hide the visibility of characters in the dataform password editor. You can enable this toggle by setting the `EnablePasswordVisibilityToggle` property to true in `DataFormItem`. The default value of `EnablePasswordVisibilityToggle` is `true`.
```

Comment:
Password toggle visibility in a floating label layout is used to show or hide the visibility of characters in the dataform password editor. Enable this toggle by setting the `EnablePasswordVisibilityToggle` property to true in the `DataFormItem.` The default value of the `EnablePasswordVisibilityToggle` is `true.`

Reply:
Content changed